src/lib.rs

```diff
@@ -265,6 +282,8 @@ pub mod pallet {
 			Self::validate_fully_signed(&params, &state, state_sigs)?;
 			let channel_id = state.channel_id;
 
+			// AppRegistryFor::validTransition::<T>::();
+
```

Comment:
```suggestion
```

Reply:
Already deleted in the latest version. (I created a separate branch where I experimented with making the app identifier configurable, as this was needed for the integration in the go-perun library, where the AppID is assumed to be of type wallet.Address currently.)

Reply:
Okay thank you so much 👍

Reply:
Thanks for taking the time to review!